m4_sessions/1901-inning/EI-matters.md

```diff
@@ -18,43 +18,37 @@ Students will learn a basic framework for understanding emotional intelligence a
 ## Session Outline
 
 
- 1. Introductions (10 minutes, lead by , see slide deck for intro format )
+ 1. Introductions (15 - 20 minutes, lead by , see slide deck for intro format )
+    - Icebreaker
  2. Intro to ei - Description and surface level explanation. (5 minutes, slides 3-6, lead by )
- 3. Article reading and discussion (20 minutes)
-     - We'll assign three groups. Each group gets 5 minutes to read one of the following articles:
-          1. [13 Signs of High Emotional Intelligence](https://www.inc.com/justin-bariso/13-things-emotionally-intelligent-people-do.html)
+ 3. Video and discussion (20 minutes)
+    - 3 - 5 minute video
```

Comment:
Which video?

Reply:
https://www.youtube.com/watch?v=LgUCyWhJf6s for the moment